pkg/client/applyconfiguration/minio.min.io/v2/statefulsetmetadata.go

```diff
@@ -0,0 +1,60 @@
+// This file is part of MinIO Operator
+// Copyright (c) 2024 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// Code generated by applyconfiguration-gen. DO NOT EDIT.
+
+package v2
+
+// StatefulSetMetadataApplyConfiguration represents an declarative configuration of the StatefulSetMetadata type for use
```

Comment:
```suggestion
// StatefulSetMetadataApplyConfiguration represents a declarative configuration of the StatefulSetMetadata type for use
```

Reply:
Automatically generated, so can't do anything about it.